utilities/ovs-ofctl.c

```diff
@@ -4169,6 +4169,40 @@ ofctl_update_map(struct ovs_cmdl_context *ctx)
     free(value);
 }
 
+static void
+ofctl_dump_map(struct ovs_cmdl_context *ctx)
+{
+    struct ofpbuf *request;
+    struct ofpbuf *reply;
+    struct vconn *vconn;
+    enum ofputil_protocol protocol;
+    enum ofputil_protocol usable_protocols = OFPUTIL_P_OF13_UP | OFPUTIL_P_OF10_NXM;
+    enum ofp_version version;
+
+    const char *bridge = ctx->argv[1];
+    const ovs_be16 prog = atoi(ctx->argv[2]);
+    const ovs_be16 map_id = atoi(ctx->argv[3]);
+    int verbosity = 3;
+    if (ctx->argc > 4) {
+        if (strcmp(ctx->argv[4], "hex") == 0) {
+            verbosity = 0;
+        }
+    }
```

Comment:
If `verbosity` is replaced with a boolean `hex` variable, then this can be replaced with:
```c
hex = strcmp(ctx->argv[4], "hex") == 0;
```